spec/features/visitor_can_view_categories_page_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+describe 'Visitor can view category pages', type: :feature do
+  scenario 'visitor clicks link to category page from homepage' do
+    category_1 = FactoryGirl.create(:category)
+    category_2 = FactoryGirl.create(:category)
+    # As a visitor
+    # when I visit the home page
+    visit farmersmarket_path
+    # and I click on the categories dropdown
+    # I see a list of categories
+    expect(page).to have_select("Categories", options: [category_1.name, category_2.name])
+    # when I select a category
+    select "#{category_1.name}", from: 'Categories'
+    # I am taken to the category page
+    expect(current_path).to eq category_path(category_1)
+    expect(page).to have_content("#{category_1.name}")
+
+    select "#{category_2.name}", from: 'Categories'
```

Comment:
Oh, I know. I can see how that may have been taken as a criticism but it was more of a , "kudos!"